package.json

```diff
@@ -79,11 +79,11 @@
     "node-gyp": "^9.3.0"
   },
   "scripts": {
-    "build": "rollup -c build-settings/rollup.config.js",
-    "build:prodsizecheck": "rollup -c build-settings/rollup.config.js --configPlatforms='browser' --configFormats='esm' --configEnvironment='production'",
+    "build": "rollup -c build-settings/rollup.config.mjs",
+    "build:prodsizecheck": "yarn rollup --configPlatforms='browser' --configFormats='esm' --configEnvironment='production'",
     "build:types": "yarn tsc --build --verbose tsconfig-build.json",
     "build:docs": "typedoc",
-    "watch": "rollup -c build-settings/rollup.config.js --watch",
+    "watch": "yarn rollup --watch",
```

Comment:
Why don't we need to pass `rollup` the location of the config file?

Reply:
Because it's running the `rollup` yarn script that already passes it.

Reply:
I don't see a script called "rollup".

Reply:
🤦🏻 Yup, I never pushed that commit - it's still called `build` in this version. Doh! Thanks for catching that.

Reply:
Pushed.